Document NPM Packages (#13)
eik-lib/eik-lib.github.io: docs/server_file_structure.md

The file structure of a package is stored in a package file at `/:root/:org/pkg/:name/:version.package.json`.

## NPM Packages

> NPM packages are packages found in the NPM registry and published to Eik as a "copy". Packages from

Reviewer: ```suggestion
NPM packages are packages from the NPM registry that are then published to Eik as a "copy". Packages from
```

> the NPM registry is published under this namespace to avoid name collision with packages.

Reviewer: ```suggestion
the NPM registry are published under this namespace to avoid name collision with other packages.
```

NPM packages are stored under `/:root/:org/npm/:name/:version/` and the structure of a package is

> arbitrary and untouched during upload by the service.

Reviewer: ```suggestion
arbitrary and is not changed during upload by the service.
```

> The file structure of a NPM package is stored in a package file at `/:root/:org/pkg/:name/:version.package.json`.

Reviewer: ```suggestion
The file structure of an NPM package is stored in a package meta file at `/:root/:org/pkg/:name/:version.package.json`.
```
